use-timescale/migration/about-hypershift.md

```diff
@@ -0,0 +1,119 @@
+---
+title: About migrating to Timescale Cloud with hypershift
+excerpt: Migrate an existing PostgreSQL database to Timescale Cloud in a single step
+products: [cloud]
+keywords: [data migration, Hypershift]
+tags: [ingest, Hypershift, postgresql]
+---
+
+import HypershiftIntro from "versionContent/_partials/_hypershift-intro.mdx";
+
+# About Hypershift
+
+<HypershiftIntro />
+
+## Migrating from a managed database
+
+If you're migrating from an Amazon RDS database, the best option is to run
+hypershift on an EC2 instance in the same availability zone as your RDS
+instance.
+
+<Highlight type="note">
+Availability zones are assigned different names in different AWS
+accounts. To make sure that you are using the same zone refer to
+[AWS user guide](https://docs.aws.amazon.com/ram/latest/userguide/working-with-az-ids.html)
+</Highlight>
```

Comment:
I don't think that we should include this. It's not actionable by the user, so only adds confusion.

Reply:
```suggestion
```